plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -88,12 +105,17 @@ bool EuroliteProFactory::DeviceAdded(
   // Eurolite USB-DMX512-PRO MK2?
   } else if (descriptor.idVendor == VENDOR_ID_MK2 &&
              descriptor.idProduct == PRODUCT_ID_MK2) {
-    if (m_enable_eurolite_mk2) {
-      OLA_INFO << "Found a possible new Eurolite USB-DMX512-PRO MK2 device";
-      LibUsbAdaptor::DeviceInformation info;
-      if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
-        return false;
-      }
+    LibUsbAdaptor::DeviceInformation info;
+    if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
+      return false;
+    }
+
+    const bool serial_matches =
+      STLContains(m_expected_eurolite_mk2_serials, info.serial);
+
+    if (m_enable_eurolite_mk2 || serial_matches) {
+      OLA_INFO << "Found a possible new Eurolite USB-DMX512-PRO MK2 device "
+                  "with serial " << info.serial;
```

Comment:
I don't know if it's worth logging two different messages here (especially given we've already done the "hard" work of checking for a match), then we could see from the log how it had matched (general or specific serial).

Reply:
Sure, done, have a check if this is what you mean...

Reply:
Looks good thanks